test/mento.test.ts

```diff
@@ -0,0 +1,342 @@
+import { Contract, utils } from 'ethers'
```

Comment:
Totally optional: co-locating the tests with their files in the same folder is more common in JS these days and can help keep things organized as projects grow.

Reply:
Yeah I also liked that +1